addons/addon-base/packages/services/lib/aws/aws-service.js

```diff
@@ -87,7 +87,7 @@ class AwsService extends Service {
    * @returns {Promise<{accessKeyId, secretAccessKey, sessionToken}>}
    */
   async getCredentialsForRole({ roleArn, roleSessionName, externalId }) {
-    const sts = new this.sdk.STS({ apiVersion: '2011-06-15' });
+    const sts = new this.sdk.STS({ apiVersion: '2011-06-15', endpoint: 'https://sts.ap-east-1.amazonaws.com' });
```

Comment:
Instead of hardcoding `ap-east-1`, you can add this instead `stsRegionalEndpoints: 'regional'`